antelope_contracts/contracts/deposit_proxy/README.txt

```diff
@@ -0,0 +1,12 @@
+--- deposit_proxy Project ---
+
+ - How to Build -
+   - cd to 'build' directory
+   - run the command 'cmake ..'
+   - run the command 'make'
```

Comment:
There is no need for this since it will build as part of the main build at the root of this repo, which we already have a README for. In fact, I think you can get rid of this entire README file.